### PR TITLE
feat(pie-spinner): DSW-1430 init pie spinner

### PR DESCRIPTION
**Pull request by raoufswe**
## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by kevinrodrigues — APPROVED** [review]

File: packages/components/pie-spinner/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "test:visual": "run -T cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_PIE_SPINNER} percy exec --allowed-hostname cloudfront.net -- npx playwright test -c ./playwright-lit-visual.config.ts",
+    "test:visual:ci": "yarn test:visual"
+  },
+  "author": "Just Eat Takeaway.com - Design System Team",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@justeattakeaway/pie-components-config": "workspace:*"
+  },
+  "dependencies": {
+    "@justeattakeaway/pie-webc-core": "workspace:*"
```
[the remark this answers is not in the document]
Comment (raoufswe) [reply packages/components/pie-spinner/package.json:34]: i think @siggerzz is looking into that.

File: packages/components/pie-spinner/test/visual/pie-spinner.spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (xander-marjoram) [inline packages/components/pie-spinner/test/visual/pie-spinner.spec.ts:1]: Are all these empty lines at the start of files because of the generator?

**Review by xander-marjoram — DISMISSED** [review]

File: packages/components/pie-spinner/test/visual/pie-spinner.spec.ts (hunk 2 above)
Comment (raoufswe) [reply packages/components/pie-spinner/test/visual/pie-spinner.spec.ts:1]: yes, didn't touch them :,)

**Review by thejfreitas — APPROVED** [review]

File: packages/components/pie-spinner/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (siggerzz) [reply packages/components/pie-spinner/package.json:34]: If you could update this to the latest published version for now, it'd be appreciated. It'll prevent any failures when this package is installed outside the monorepo :)

Same applies for the config package above

[the remark this answers is not in the document]
Comment (raoufswe) [reply packages/components/pie-spinner/package.json]: I updated the current references manually and created a ticket  `DSW-1726` to update the generator.

**Review by siggerzz — APPROVED** [review]

File: packages/components/pie-spinner/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+</p>
+
+# Table of Contents
+
+1. [Introduction](#pie-spinner)
+2. [Installation](#installation)
+3. [Importing the component](#importing-the-component)
+4. [Peer Dependencies](#peer-dependencies)
+5. [Props](#props)
+6. [Testing](#testing)
```
Comment (kevinrodrigues) [inline packages/components/pie-spinner/README.md:18]: Do we need a section for this or does the generator need updating?

**Review by LTurns — APPROVED** [review]

File: packages/components/pie-spinner/README.md (hunk 3 above)
Comment (raoufswe) [reply packages/components/pie-spinner/README.md:18]: i will check this in a follow up PR

**Pull request merged**